SavingsManagementSystem.Service/Authentication/Implementations/VerificationTokenService.cs

```diff
@@ -14,14 +14,16 @@ public VerificationTokenService(IUnitOfWork unit)
 			_unit = unit;
 		}
 
-		public async Task<VerificationToken> CreateVerificationTokenAsync(string userId, int expiryMinutes)
+		public async Task<VerificationToken> CreateVerificationTokenAsync(int expiryMinutes, string status, string email, string? userId = null)
 		{
 			var createdOn = DateTime.UtcNow;
```

Comment:
remove this and set date directly when sending it

Reply:
ExpiryTime = createdOn.AddMinutes(expiryMinutes)
i needed to add the minute to createdOn, thats why im only requesting for the minute

		var createdOn = DateTime.UtcNow;
			var vToken = new VerificationToken
			{
				Id = Guid.NewGuid().ToString(),
				UserId = userId,
				Token = Guid.NewGuid().ToString(),
				Status = status,
				Email = email,
				IsUsed = false,
				CreatedOn = createdOn,
				ExpiryTime = createdOn.AddMinutes(expiryMinutes)
			};